features/api/v1/engines/oci/manifests.feature

```diff
@@ -0,0 +1,1580 @@
+      # alpine 3.20.3
+      | 90fa9487-49ea-487a-a47c-83e9f60d1651 | 14c038fd-b57e-432d-8c09-f50ebcd6a7bc | 5762c549-7f5b-4a73-9873-3acdb1213fe8 | 757e0a41-835e-42ad-bad8-84cabd29c72a | index.json                                                                    | application/vnd.oci.image.index.v1+json                   | sha256:355eee6af939abf5ba465c9be69c3b725f8d3f19516ca9644cf2a4fb112fd83b | 441            | {"schemaVersion":2,"mediaType":"application/vnd.oci.image.index.v1+json","manifests":[{"mediaType":"application/vnd.docker.distribution.manifest.list.v2+json","digest":"sha256:beefdbd8a1da6d2915566fde36db9db0b524eb737fc57cd1367effd16dc0d06d","size":1853,"annotations":{"containerd.io/distribution.source.docker.io":"library/alpine","io.containerd.image.name":"docker.io/library/alpine:latest","org.opencontainers.image.ref.name":"latest"}}]}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | 2024-11-11T10:26:01.000Z | 2024-11-11T10:26:01.000Z |
```

Comment:
Need to drop `index.json` manifests since they aren't meant to be served by a registry (oops).

(Fixed in 50bb888841cca62eac476152ba3d743490c2af0b.)